### ISSUE TITLE
Improvements in TimeLapse 

### ISSUE THREAD

**Issue by chepo92** [Billwilliams1952/PiCameraApp#8]
Such a great app, have you done any recent dev on Timelapse Code?, I was looking at the code and found the timelapse tab and the ToDo's

**Comment by Billwilliams1952** [comment]
Yes - it’s in the works. I haven’t had time to work on it since I’ve been busy with microVIEW and a new app called microLogger. 

Will get back to PiCamera soon. 

Bill

**Comment by OlafVal** [comment]
Hey, I am very excited about the PiCameraApp with its extensive interface and I would also be very interested in the time-lapse function.

**Comment by chepo92** [comment]
@Billwilliams1952 you took your time to reply so I developed some functions and the timelapse on this app maybe i'll push to your repo, I didn't know if you were still active
